scripts/prepare-for-platform.sh

```diff
@@ -9,6 +9,16 @@ NC='\033[0m'
 PLATFORM=""
 PLATFORM_FOLDER=""
 
+DO_SPACE_URL=https://status-go.ams3.digitaloceanspaces.com
+GITHUB_URL=https://github.com/status-im/status-go/releases
+STATUS_GO_VER="$(cat STATUS_GO_VERSION)"
+
+ANDROID_LIBS_DIR="android/app/libs"
+STATUS_GO_DRO_ARCH="${ANDROID_LIBS_DIR}/status-go-${STATUS_GO_VER}.aar"
+
+RCTSTATUS_DIR="modules/react-native-status/ios/RCTStatus"
+STATUS_GO_IOS_ARCH="${RCTSTATUS_DIR}/status-go-ios-${STATUS_GO_VER}.zip"
+
 #if no arguments passed, inform user about possible ones
 
 if [ $# -eq 0 ]; then
```

Comment:
The message in this if is outdated:
https://github.com/status-im/status-react/pull/7818/files#diff-6bd7802fe8ef4fa7420bc038a4de9887R27